sdk/src/dataMapper/ModuleDataMapper.ts

```diff
@@ -107,7 +107,7 @@ export default class ModuleDataMapper extends BaseDataMapper<Module, Module_filt
   }
 
   async getModulesNumber() {
-    return await this.executeReadMethod("getModulesNumber", []);
+    return await super.findTotalCount();
```

Comment:
I don't think we need to keep this `await` here.
Actually, I think we can remove a lot of unnecessary `await` in this file!